test/ppxlib-pp-ast/basic.t

```diff
@@ -48,7 +53,10 @@ But it can also be used to pretty a single expression:
   Pexp_apply
     ( Pexp_ident (Lident "+")
     , [ ( Nolabel, Pexp_ident (Lident "x"))
-      ; ( Nolabel, Pexp_constant (Pconst_integer ( "2", None)))
+      ; ( Nolabel
+        , Pexp_constant
+            { pconst_desc = Pconst_integer ( "2", None); pconst_loc = __loc}
```

Comment:
We should probably add a special case to `Pp_ast` here as well, so that unless specified otherwise, the `pconst_desc` is printed directly, without the wrapping record.